config/environments/development.rb

```diff
@@ -19,7 +19,7 @@
   # Don't care if the mailer can't send
   config.action_mailer.raise_delivery_errors = false
 
-  # Print deprecation notices to the Rails logger
+  # Print deprecation notices to the Rails logger.
```

Comment:
Man, this line is _**SUPER**_ important!

Reply:
> Man, this line is _**SUPER**_ important!

Period.